### DISCUSSION TITLE
Handy tool for those with multiple remotes

### DISCUSSION THREAD

**Discussion by pfuller88** [GerryDazoo/Slinger#352] [General]
I created a website which I self host that has all the links to the different remotes I use.  It has two sections, one for localhost and one for web addresses.  When you click on a button it will open a new remote window, to the left with set width and heigh (you can change all these in the java function in the html.   Also, in the links, change the "yourdomain" to your domain AND the at the end of the link, change the name to your unified ini singbox setion name  - i.e. in the sample I have... 

http://www.yourtdomain.com:8080/Remote/500max'.

Change the "yourdomain" and the "500max".  Change the port as needed and ".com" if your domain is different.

You will need a web server to host this website, i.e. apache.

[remote links website.zip](https://github.com/GerryDazoo/Slinger/files/10086540/remote.links.website.zip)

**Comment by sp222222** [comment]
Neato , do you have screenshots of this? And which version of ss are you using? 3.07? 3.08x?

**Reply by pfuller88** [reply]
Latest version, that was released yesterday, or so.  Remember, the code above is hosted on your own webserver, not within slinger.   Here is a screenshot.
![image](https://user-images.githubusercontent.com/117112102/204094865-9c62069f-5c17-47ec-a7e9-5d1d5d5f102a.png)

**Reply by sp222222** [reply]
beautiful. QQ - I'm a tightwad and still use frontpage'98 to maintain my webpage (it's pics of my family and weblinks to things I like to get to all the time). Is there a free webpage editor out there you suggest? My FP98 keeps crashing when trying to open the html itself, but I was able to add the page/link to my main page as a hyperlink. I was hoping your zip would provide the nine buttons you have shown in the screenshot. it would then be a quick mod to pointing each button to the correct slingbox and/or speed/audio needed. thanks !

**Reply by pfuller88** [reply]
only one i can think of is AWS free tier.   it is a little cumbersome to setup and stay within the free tier limits. Would need to setup the free tier then install ubuntu server, then LAMP server on that.   outside of that your on your own.

**Reply by sp222222** [reply]
thx lol servers - i'm running windows here :)

**Reply by GerryDazoo** [reply]
mikey. You can run apache on Windows. That's where home.dazoo.net resides.

**Reply by GerryDazoo** [reply]
many moons ago someone was also pitching an Oracle free cloud server.

**Reply by sp222222** [reply]
http://bluegriffon.org/#download pfuller88 is your code CSS or HTML? I am going to try to edit your webpage with this free one here

**Reply by pfuller88** [reply]
It is HTML with some java script.  CSS are style files for HTML code.

**Reply by sp222222** [reply]
I got it working on my website editing it with bluegriffon.org and publishing with frontPage - thanks !

**Reply by Toei79** [reply]
curious if it can work on nginx?
i have a local server nginx, guessing yes right? just too new on this.
